api/jsonapi.go

```diff
@@ -4150,3 +4153,62 @@ func (i *jsonAPIHandler) GETPost(w http.ResponseWriter, r *http.Request) {
 	}
 	SanitizedResponseM(w, out, new(pb.SignedPost))
 }
+
+// POSTSendOrderMessage - used to manually send an order message
+func (i *jsonAPIHandler) POSTSendOrderMessage(w http.ResponseWriter, r *http.Request) {
+	type sendRequest struct {
+		OrderID     string `json:"orderID"`
+		MessageType int32  `json:"messageType"`
```

Comment:
MessageType should be a string instead of the internal enum we use to identify a message type. (The internal enum is an implementation detail of the protocol that we shouldn't expose if we don't have to.) Verbatim strings for each message type would be fine for now.